docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -135,16 +163,37 @@ cluster, and the summarization part will affect the status summarizer.
 ## Design overview
 
 In very brief: the design is to reduce each edge placement problem to
-many instances of kcp's TMC problem.
+many instances of kcp's TMC problem.  Part of making many instances of
+the TMC problem is to maintain a unique _mailbox workspace_ in the
+center for each edge cluster.
+
+However: to fully realize the goals of this PoC we will have to use a
+modified version of TMC, which differs from today's TMC in ways
+summarized below in the [Syncers](#syncers) section.
+
+As in TMC, in this design we have _downsync_ and _upsync_ --- but they
+are a little more complicated here.  Downsync involves propagation of
+desired state from workload management workspace through mailbox
+workspaces to edge and return/summarization of reported state.  Upsync
+involves return/summarization of desired and reported state of objects
```

Comment:
Doesn't it also include the upsync of the schema itself, before we upsync the resources themselves ?

Reply:
Not necessarily. The resource definitions might exist in the center as consequences of CRDs or APIBindings, for example, as discussed elsewhere.